mkt/search/tests/test_views.py

```diff
@@ -200,6 +200,11 @@ def test_right_category(self):
         eq_(res.status_code, 200)
         eq_(res.json['objects'], [])
 
+    def test_old_category(self):
+        res = self.anon.get(self.url, data={'cat': 'books'})
+        eq_(res.status_code, 200)
+        eq_(res.json['objects'], [])
```

Comment:
We should also add a test to make sure the old category name properly returns content from the new one.